uwblueprint/planet-read
# Run frontend on IONOS

## Notion ticket link



[Run frontend on IONOS](https://www.notion.so/uwblueprintexecs/Run-frontend-on-IONOS-784cbebb1d4a4b5f92999315e8056c38)



## Implementation description

- Set up frontend to display at [https://www.planetread.org/aml_platform](https://www.planetread.org/aml_platform) *it's running an older vers of main
- NOTE before merging+tagging: will have to re-init git ftp because I pushed to it while testing this branch



## Steps to test

1. Poke around [https://www.planetread.org/aml_platform](https://www.planetread.org/aml_platform) and make sure things look okay
2. Poke around locally and make sure things look okay
3. Poke around on firebase preview and make sure things look okay



## What should reviewers focus on?

- Do the routes all work properly?

## Checklist
